timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/tripleprocessor/CollectionMembershipTripleProcessor.java

```diff
@@ -16,6 +16,7 @@ public void process(String vreName, boolean isAssertion, Triple triple) {
 
     if (isAssertion) {
       entity.addToCollection(database.findOrCreateCollection(vreName, triple.getObject()));
+      entity.removeFromCollection(database.getDefaultCollection(vreName));
```

Comment:
yes, the last commit is made superfluous, but I would recommend retaining the capability of copying properties from any existing collection for now, because I am pretty sure that at one point we will want to support multiple rdf types as a feature